stable/external-dns/values.yaml

```diff
@@ -195,23 +209,9 @@ coredns:
 
 ## OpenStack Designate provider configuration to be set via arguments/env. variables
 ##
-designate:
-  ## Use a custom CA (optional)
-  ##
-  customCA:
-    enabled: false
-    ## The content of the custom CA file
-    ##
-    content: ""
-    ## Location to mount custom CA
-    ##
-    mountPath: "/config/designate"
-    ## Custom CA filename
-    ##
-    filename: "designate-ca.pem"
-
+designate: {}
   ## Use a host path custom CA (optional)
-  ##  This conflicts setting the above customCA to true and chart rendering will fail if you set customCA to true and specify customCAHostPath
+  ##  This conflicts with global customCA settings. Chart rendering will fail if you set customCA to true and specify customCAHostPath
```

Comment:
These changes differ from the ones I see in the **values-production.yaml**. Please ensure they're sync

Reply:
Thanks @juan131.

This should've been done in a different PR that brought in extra Designate settings #20746
I have updated that, and also rebased, fixed author name, and signoffs.